examples/decoding/plot_decoding_spatio_temporal_source.py

```diff
@@ -91,14 +91,14 @@
 scores = cross_val_multiscore(time_decod, X, y, cv=7, n_jobs=1)
 
 # Plot average decoding scores of 10 splits
```

Comment:
We generally recommend between 3 and 10 splits when using a partition CV. I can reduce the number for speed purposes if you want